SessionMessagingKit/Sending & Receiving/MessageSender.swift

```diff
@@ -433,15 +433,16 @@ public final class MessageSender {
             )
             .done(on: DispatchQueue.global(qos: .default)) { responseInfo, data in
                 message.openGroupServerMessageId = UInt64(data.id)
-
+                let serverTimestampMs: UInt64? = (data.posted == nil) ? nil : UInt64(floor(data.posted! * 1000))
```

Comment:
So you should be able to clean this up (and avoid the force-unwrap) by mapping from the nullable type:
```
let serverTimestampMs: UInt64? = data.posted.map { UInt64(floor($0 * 1000)) }
```